docs/DeveloperGuide.adoc

```diff
@@ -544,6 +544,58 @@ Use case resumes at step 1.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: UC09: Browse
```

Comment:
Any possible extensions here? E.g. no Internet access

Also, a suggestion for the use case title:
```suggestion
=== Use case: UC09 - Browse the web
```